source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/projects/show/overview/reviewers/form.tsx

```diff
@@ -50,6 +57,19 @@ class AddReviewerForm extends React.Component<IProps> {
     });
   };
 
+  setEmptyLocale = () => {
+    const { currentUser, i18n } = this.props;
+    const { locale } = this.state;
+    var defaultLocale = locale;
```

Comment:
let should be used here instead of `var`.
var has scoping issues https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let